testcases/TC06.2-pass-1.md

```diff
@@ -50,11 +50,10 @@ sample: /testfiles/TF06/06.2-all-pass-1.html
 ---
 | Test Instruction | Instruction Detail | Expected Test Case Result |
 |------------------|--------------------|---------------------------|
-| **IC-1** | Identify any image that conveys decoration | One `<img>` exists in the test data code sample. The image is pure decoration. |
-
-| **6.2-1** | Check to see that the ARIA role is "presentation". | PASS |
-| **6.2-2** | Check to see that the ARIA role is "none". | Not applicable |
-| **6.2-3** | Check to see that the aria-hidden is set to "true".| Not applicable |
-| **6.2-4** | Check to see that the text alternative (combination of accessible name and accessible description) is empty. | Not applicable |
-| **6.2-5** | Check to see that the image is inserted via CSS. | Not applicable |
-| **Result** | If any of the above checks fail, then Baseline Test 6.2-DecorativeImage fails. | PASS. |
+| **IC-1** | Identify any decorative image that is pure decoration, is used only for visual formatting, or is not presented to users. |
+
+Check that at least one of the following is true [SC 1.1.1]:
```

Comment:
this row will need to be within the table format, so | |Check that at least one of the following is true [SC 1.1.1]:| | is probably what's needed.